### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by rdviii**
Today's work on improving design documents.

### CODE REVIEW

**Comment by Chibikuri** [comment]
Can I approve this  pull request?

**Comment by rdviii** [comment]
Yes!

**Comment by rdviii** [comment]
Although you should "review" it first.

File: doc/Beam_splitter.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+they represent the polarization of photon in a particular place and at
+a fixed time.  Weihs and Zeilinger refer to it as spin representing an
+_additional_ degree of freedom, and are very explicit about writing
+out the behavior of the BS in four bases: H and V at each of the two
+input ports.  This allows the unitary to be written properly, with the
+after state being a four-basis representation at the two output ports.
+
+Based on Eqs. 1.21 & 1.21 of W&Z, in the basis
+
+$$\left(\begin{matrix}
```
Comment (Chibikuri) [inline doc/Beam_splitter.md]: These latex texts can't be recognized in Github markdown. You may have to use third-party tools to shot it correctly.

File: doc/Classical Chassis Control Card (C4).md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Classical Chassis Control Card (C4) #
+
+*Note: This is an old design document, which we expect to still be useful, but it may or may not correspond well to the current implementation.
+
+In a [Router](Router%20Architecture.md), the classical control hardware module where most of the [Router Software Architecture](Router Software Architecture.md) resides. A [Repeater](Repeater.md) should have one, too.
```
Comment (Chibikuri) [inline doc/Classical Chassis Control Card (C4).md]: Links are expired.

File: doc/Congestion Collapse.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Research question: Congestion Collapse #
+
+Does the quantum equivalent of congestion collapse exist? (We have a strong intuition that the answer here is "yes" for anything other than circuit switching, but we need to characterize it.)
+
+This is the macro-level problem, for the network as a whole. It is somewhat related to the incast problem under [Competition for Resources](Competition%20for%20Resources.md).
```
Comment (Chibikuri) [inline doc/Congestion Collapse.md]: This link also doesn't work.

File: doc/Connection Creation.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+Connection creation consists of three components:
+
+- Path selection ([[Routing]])
+- [[Resource Allocation]]
+- [[Role Assignment]]
+
+We will begin with flat versions of all three, at least for pedagogical purposes, then extend to recursive.
+
+Once a connection is created, of course, we want to operate it. See [Repeater Operation](Repeater Operation) and [Router Operation](Router Operation).
```
Comment (Chibikuri) [inline doc/Connection Creation.md]: Markdown broken

File: doc/Distributed Tomography and State Monitoring.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+preparation process ("process tomography").  See Kwiat for basic
+tutorial on full tomography, recent papers by Robin Blum-Kohout for
+valuable shortcuts.
+
+We need to do this in a distributed fashion, which is a process that
+no one seems to really fully understand yet. See rdv's upcoming paper,
+"Distributed Management of Density Matrices," for some thoughts on how
+to do this.
+
+See the [God Channel](God%20Channel.md) for a discussion of the
```
Comment (Chibikuri) [inline doc/Distributed Tomography and State Monitoring.md]: Pages don't work

File: doc/EndNode Architecture.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Quantum End Node Architecture #
+
+*Note: This is an old design document, which we expect to still be useful, but it may or may not correspond well to the current implementation.
+
+An end node has exactly one interface.  It has some similarities to a
+[QNIC](QNIC.md) augmented with a [C4](Classical%20Chassis%20Control%20Card%20(C4).md), except that it has only
```
Comment (Chibikuri) [inline doc/EndNode Architecture.md]: Also

File: doc/FlatPathSetupReturn Message Contents.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+This is the "flat" version, without recursion.
+
+* ConnectionSpec
+    * InitiatorAddr address (format unspecified)
+    * ResponderAddr address (format unspecified)
+    * (any other identifying information? port #, etc.?)
+    * TargetFidelity
+* RuleSetStack
+    * ordered list of [RuleSets](RuleSet)
```
Comment (Chibikuri) [inline doc/FlatPathSetupReturn Message Contents.md]: Here

File: doc/Fowler Path Setup.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+*Note: this file is an early design document, and these 2G networks
+are not yet handled.  Internetworking (recursive) connections and
+networks are not yet handled.*
+
+*(obviously, also, this document was left incomplete)*
+
+This is the surface code communication path setup algorithm, designed
+for use as part of the full recursive Quantum Internet.
+
+This procedure should create a set of rules like those in [Fowler
```
Comment (Chibikuri) [inline doc/Fowler Path Setup.md]: here

File: doc/HE_OBSA.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+Rather than model beam splitters and photon detectors explicitly, we
+will treat them as a unit here, and call it the BSA or OBSA.
+
+For this simulation, without loss of generality, we are assuming that
+inter-node entanglement is always generated using an OBSA.
+
+* As a [standalone BSA node](BSA Node.md), appears in an [MIM
+  Link](MIM_Link.md).
+* As a component of a node, appears once in [MM Link](MM_Link.md) and
```
Comment (Chibikuri) [inline doc/HE_OBSA.md]: here

**Pull request merged**